sr_robot_launch/scripts/sr_ur_release

```diff
@@ -0,0 +1,193 @@
+
+    def get_robot_mode(self, arm):
+        try:
+            get_mode_service = rospy.ServiceProxy("/" + arm + "_sr_ur_robot_hw/dashboard/get_robot_mode", GetRobotMode)
+            get_mode_msg = get_mode_service()
+            return get_mode_msg.robot_mode.mode
+        except rospy.ServiceException, e:
+            rospy.logerr("Service call to 'get_robot_mode' failed for arm %s. %s", arm, e)
+
+    def startup_arms(self, arms):
+        for arm in arms:
+            self.call_dashboard_service(arm, "power_on")
+        rospy.sleep(5)
+        for arm in arms:
+            self.call_dashboard_service(arm, "brake_release")
+        rospy.sleep(5)
+
+    def load_external_control_program(self, arm):
+        serv_call = rospy.ServiceProxy("/" + arm + "_sr_ur_robot_hw/dashboard/load_program", Load)
+        resp = serv_call(self.external_control_program_name)
+        print resp
+
+    def check_arms_e_stops(self, arms):
+        for arm in arms:
+            safety_mode = self.get_safety_mode(arm)
+            if safety_mode == SafetyMode.ROBOT_EMERGENCY_STOP:
+                while safety_mode == SafetyMode.ROBOT_EMERGENCY_STOP:
+                    rospy.logwarn("Emergency stop button is still pressed for arm %s, please release", arm)
+                    safety_mode = self.get_safety_mode(arm)
+                    rospy.sleep(2)
+
+    def check_arms_protective_stop(self, arms):
+        for arm in arms:
+            safety_mode = self.get_safety_mode(arm)
+            if safety_mode == SafetyMode.PROTECTIVE_STOP:
+                rospy.loginfo("Protective stop detected on: %s", arm)
+                self.call_dashboard_service(arm, "unlock_protective_stop")
+
+    def check_arms_fault(self, arms):
+        fault = False
+        for arm in arms:
+            safety_mode = self.get_safety_mode(arm)
+            if safety_mode == SafetyMode.FAULT:
+                fault = True
+                rospy.loginfo("Fault detected on: %s", arm)
+                rospy.loginfo("This can be caused by using the external e-stop. Have you released it?")
+                self.call_dashboard_service(arm, "restart_safety")
+        return fault
+
+    def check_arms_robot_mode(self, arms):
+        arm_needs_starting = False
+        for arm in arms:
+            robot_mode = self.get_robot_mode(arm)
+            if robot_mode == RobotMode.IDLE or robot_mode == RobotMode.POWER_OFF:
+                rospy.loginfo("Starting arm: %s", arm)
+                arm_needs_starting = True
+        return arm_needs_starting
+
+    def clear_arms_popups(self, arms):
+        for arm in arms:
+            self.call_dashboard_service(arm, "close_safety_popup")
+        rospy.sleep(2)
+        for arm in arms:
+            self.call_dashboard_service(arm, "close_popup")
+        rospy.sleep(2)
+
+    def get_program_state(self, arm):
+        try:
+            play_mode_service = rospy.ServiceProxy("/" + arm + "_sr_ur_robot_hw/dashboard/program_state", GetProgramState)
```

Comment:
again, just reuse generic method instead of rewrittng the string again

Reply:
df3a852d7bf5e5a16fda42d3fffb5f85ab6e8adc